source/userguide/active_learning.rst

```diff
@@ -0,0 +1,269 @@
+
+.. _al_config:
+
+Defining the AL config
+^^^^^^^^^^^^^^^^^^^^^^
+The first step towards customizing the active learning pipeline to your application is configuration adjustment. This section details different components in the configuration and provides a default configuration setup which is applied if no active learning config is defined in ``config.py``. There are five components in this configuration:
+
+1. **output_folder** (str) - Directory path to save the output of the active learning pipeline.
+
+2. **pre_tuning**
+
++---------------------------+------------+----------------------------------------------------------------------------------+
+| Configuration Key         | Type       | Definition and Examples                                                          |
++===========================+============+==================================================================================+
+| train_pattern             | str        | Regex pattern to match train files. Example for all train files ".*train.*.txt"  |
++---------------------------+------------+----------------------------------------------------------------------------------+
+| test_pattern              | str        | Regex pattern to match train files. Example for all test files ".*test.*.txt"    |
++---------------------------+------------+----------------------------------------------------------------------------------+
+| train_seed_pct            | float      | Percentage of training data to use as the initial seed                           |
++---------------------------+------------+----------------------------------------------------------------------------------+
+
+3. **tuning**
+
++---------------------------+------------+----------------------------------------------------------------------------------+
+| Configuration Key         | Type       | Definition and Examples                                                          |
++===========================+============+==================================================================================+
+| n_classifiers             | int        | Number of classifier instances to be used by ensemble strategies                 |
++---------------------------+------------+----------------------------------------------------------------------------------+
+| n_epochs                  | int        | Number of turns strategy tuning is run on the complete log data                  |
++---------------------------+------------+----------------------------------------------------------------------------------+
+| batch_size                | int        | Number of queries to sample at each iteration from the logs                      |
```

Comment:
Same comment as above. We're not operating on (un)annotated logs during the tuning phase, right?